www/themes/unl/html/SiteMaster/Core/Registry/Site/EditForm.tpl.php

```diff
@@ -9,15 +9,17 @@
             <input type="email" id="support_email" name="support_email" multiple value="<?php echo $context->site->support_email ?>" />
         </li>
         <?php if ('unl' === $context->site->group_name): ?>
-        <li>
-            <label for="support_groups">mysupport.unl.edu assignments for this site (separated by spaces and quoted if the team name includes spaces).</label>
+          <li>
+            <label for="support_groups">UNL MySupport assignments for this site.</label>
             <input type="text" id="support_groups" name="support_groups" multiple value="<?php echo $context->site->support_groups ?>" />
             <div class="help-text">
-                <p>
-                    Note that for UNLchat to create tickets in mysupport, the Support Email Address must be set to mysupport@unl.edu
-                </p>
+              <ul class="dcf-mt-1">
+                <li style="list-style-type: circle;">Separate multiple assignees by spaces and enclose in single quotes if the assignee includes spaces (i.e. 'ITS Team1' 'ITS Team2' 'ITS Team3').</li>
```

Comment:
is there another PR to handle stripping quotes and changing assignees to Primary Assignee and Other Assignees ?